### PR TITLE
Fetch content

### PR DESCRIPTION
**Pull request by CedricGuillemet**
Follow up to https://github.com/BabylonJS/JsRuntimeHost/pull/39 
- Disabling ARC by default here because of PG and bgfx calls retain/release
- clean up PG gradle
- JSE names to be same as JSR
- update 2 VT PGs
- remove `npm i` for android ci
- removed `npm i` in readme

in progress: install update

### CODE REVIEW

File: Apps/Playground/Scripts/config.json [hunk 1]
```diff
@@ -183,7 +183,7 @@
         },
         {
             "title": "LightFalloff",
-            "playgroundId": "#20OAV9#201",
+            "playgroundId": "#20OAV9#11218",
             "referenceImage": "lightFalloff.png"
         },
```
Comment (bghgary) [inline Apps/Playground/Scripts/config.json:188]: This doesn't seem to match what's in Babylon.js anymore. Should we update it?

File: CMakeLists.txt [hunk 2]
```diff
@@ -2,21 +2,32 @@
...
-FetchContent_MakeAvailable(jsruntimehost)
+
+set(CONTENT_TO_FETCH cmake-extensions jsruntimehost)
+
+if(IOS)
+    FetchContent_MakeAvailable(ios-cmake)
+    set(CMAKE_TOOLCHAIN_FILE "${ios-cmake_SOURCE_DIR}/ios.toolchain.cmake" CACHE PATH "")
+    set(PLATFORM "OS64COMBINED" CACHE STRING "")
+    set(DEPLOYMENT_TARGET "12" CACHE STRING "")
+    set(ENABLE_ARC OFF CACHE STRING "Enables or disables ARC support.")
```
Comment (bghgary) [inline CMakeLists.txt:29]: Why turn this off by default? If bgfx needs this to be off, I think we should only set it off for bgfx.

**Comment by CedricGuillemet** [comment]
Duplicate with https://github.com/BabylonJS/BabylonNative/pull/1260

File: Apps/Playground/Scripts/config.json (hunk 1 above)
Comment (CedricGuillemet) [reply Apps/Playground/Scripts/config.json:188]: A new pass on tests should be done. Some are updated in bjs and not here. some are replaced,...

File: CMakeLists.txt (hunk 2 above)
Comment (CedricGuillemet) [reply CMakeLists.txt:29]: I tried without success. if I force the flag for bgfx, then the brige is failing to compile. in the doc, the command line disables arc : https://github.com/BabylonJS/BabylonNative/blob/master/BUILDING.md#building-on-macos-targeting-ios

**Pull request closed without merging**